ee/management/commands/setup_test_environment.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+
+from posthog.ee import is_ee_enabled
+
+
+class Command(BaseCommand):
+    help = "Set up databases for non-Python tests that depend on the Django server"
+
+    def handle(self, *args, **options):
+        if is_ee_enabled():
+            from ee.clickhouse.clickhouse_test_runner import ClickhouseTestRunner as TestRunner
+        else:
+            from django.test.runner import DiscoverRunner as TestRunner
+        test_runner = TestRunner(interactive=False)
+        test_runner.setup_databases()
+        test_runner.setup_test_environment()
```

Comment:
In https://github.com/PostHog/plugin-server/pull/34/files, where we need to set up both Postgres AND ClickHouse for new ingestion pipeline's tests. At first I considered just rewriting the setup in JS… but damn, what a mess that would be. Especially considering that the Django server is the source of truth and we need to keep ingestion in sync with it.
The current state of https://github.com/PostHog/plugin-server/pull/34/files is that ALL tests run this command and get an environment just like they would in production (instead of mocks that are incomplete and can go out of sync very easily) – much simpler and works really nicely, even if CI needs to pull this repo.